qrenderdoc/Code/Interface/Extensions.h

```diff
@@ -927,6 +927,79 @@ an unknown option is passed, nothing will happen.
 )")
   virtual void SelectComboOption(QWidget *combo, const rdcstr &option) = 0;
 
+  DOCUMENT(R"(Create a progress bar widget.
+
+By default the progress bar has minimum and maximum values of 0 and 100. These can be changed with
+:meth:`SetProgressBarRange`.
+
+:param bool horizontal: the progress bar orientation, true for horizontal otherwise vertical.
+:return: The handle to the newly created widget.
+:rtype: QWidget
+
+)")
+  virtual QWidget *CreateProgressBar(bool horizontal) = 0;
+
+  DOCUMENT(R"(Reset a progress bar widget.
+
+The progress bar "rewinds" and shows no progress. The minimum and maximum values are not changed.
```

Comment:
I know this is a quote from the Qt documentation, but IMO it doesn't do a good job of explaining what this actually means. I went to check if this was meaningfully different from setting a value of 0 and I see that on some themes the progress bar will have a % text label beside it, and while `setValue(0)` will show 0% `reset()` will hide the text. It would be good to state that for people.

Reply:
Done. Thanks for letting me know about that difference.